helper_win.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+"""
+A Python script that attempts to check that the configuration of your
+nativeMessaging app is set up correctly
+
+Currently requires Python 3.6 minimum.
+
+If you find more issues with setting this up, let's see if we can add to this
+script.
+"""
+import argparse
+import json
+import os
+import subprocess
+import winreg
+
+# Command-Line
+parser = argparse.ArgumentParser(description="Helper for ff2mpv on windows.")
+group = parser.add_mutually_exclusive_group(required=True)
+group.add_argument(
+    "-c",
+    "--check",
+    action="store_const",
```

Comment:
Nitpick: Use `store_true` instead of `store_const`. Then, you can remove `const=...` and `dest=...` and your checks will be a lot more fluent:

```python
if args.check:
    print("--check")
elif args.intall:
    print("--install")
```

...and so forth.